_data/simple-icons.json

```diff
@@ -8603,7 +8642,7 @@
         },
         {
             "title": "Starling Bank",
-            "hex": "7433FF",
+            "hex": "6935D3",
```

Comment:
Where did you find this new color? 🤔

Reply:
Found in this PDF:
https://www.starlingbank.com/docs/brand/starling-bank-brand-guidelines.pdf

I also created a new issue to update this brand logo (#6143)

Reply:
Okay, thanks for clarifying :+1: